config/development.json

```diff
@@ -67,5 +67,6 @@
 	"SESSION_SECRET": "dev-session-secret",
 	"FEATURE_COURSE_SHARE": true,
 	"FEATURE_COLUMN_BOARD_ENABLED": true,
-	"FEATURE_COLUMN_BOARD_SUBMISSIONS_ENABLED": true
+	"FEATURE_COLUMN_BOARD_SUBMISSIONS_ENABLED": true,
+	"FEATURE_COLUMN_BOARD_EXTERNAL_TOOLS_ENABLED": true
```

Comment:
This env will be only used in client, right?

Reply:
this env will be used for local development